### PR TITLE
[metalkit] Add new API from both beta 1 and beta 2

### PR DESCRIPTION
**Pull request by spouliot**

### CODE REVIEW

File: src/metalkit.cs [hunk 1]
```diff
@@ -143,17 +154,52 @@ interface MTKTextureLoaderKeys {
...
+		[Field ("MTKTextureLoaderOptionTextureStorageMode")]
+		NSString TextureStorageModeKey { get; }
+
+		[iOS (10,0)][Mac (10,12, onlyOn64 : true)]
+		[Field ("MTKTextureLoaderOptionCubeLayout")]
+		NSString CubeLayoutKey { get; }
+
+		[iOS (10,0)][Mac (10,12, onlyOn64 : true)]
+		[Field ("MTKTextureLoaderOptionOrigin")]
+		NSString OptionOriginKey { get; }
```
Comment (VincentDondain) [inline src/metalkit.cs]: Should just be `OriginKey` instead of `OptionOriginKey`.

File: src/metalkit.cs [hunk 2]
```diff
@@ -143,17 +154,52 @@ interface MTKTextureLoaderKeys {
...
+	}
+
+	[iOS (10,0)][Mac (10,12, onlyOn64 : true)]
+	enum MTKTextureLoaderCubeLayout {
+		[Field ("MTKTextureLoaderCubeLayoutVertical")]
+		LayoutVertical,
```
Comment (VincentDondain) [inline src/metalkit.cs]: Layout is redundant here since the name of the enum is `MTKTextureLoaderCubeLayout`.

**Comment by VincentDondain** [comment]
LGTM 👍

**Comment by xamarin-release-manager** [comment]
Build success

**Comment by xamarin-release-manager** [comment]
Build success

**Comment by xamarin-release-manager** [comment]
Build success

**Pull request merged**